gateway/gateway.go

```diff
@@ -16,6 +16,7 @@ const (
 	delimiter                 = "<<>>"
 	eventStreamSourceCategory = "eventStream"
 	extractEvent              = "extract"
+	rETLEvent                 = "record"
```

Comment:
can record format be used by server side sdks too or do we want to restrict only to rETL?

Reply:
I would say rETL. Don't see how would it be useful for server side sdks.